wolfcrypt/src/ecc.c

```diff
@@ -5676,7 +5676,12 @@ int wc_ecc_make_key_ex2(WC_RNG* rng, int keysize, ecc_key* key, int curve_id,
     if (err == MP_OKAY) {
         err = _ecc_validate_public_key(key, 0, 0);
     }
-    if (err == MP_OKAY) {
+    if (err == MP_OKAY
+#if defined(WOLF_CRYPTO_CB && defined(WOLFSSL_HYBRID_FIPS_TPM))
```

Comment:
You've got a misplaced `()`. The logic should be `#if defined(HAVE_FIPS) && defined(WOLF_CRYPTO_CB)` and same ` && key-devId == INVALID_DEVID`. Just remove the `WOLFSSL_HYBRID_FIPS_TPM`